.github/actions/ci/action.yml

```diff
@@ -9,6 +9,13 @@ inputs:
     description: 'Version of the C++ Server-side SDK to use for building and testing.'
     required: false
     default: 'launchdarkly-cpp-server-redis-source-v2.1.0'
+  server-rockspec:
+    description: 'The rockspec file for the server-side SDK.'
+    required: true
+  redis-rockspec:
+    description: 'The rockspec file for the Redis integration'
+    required: true
+
```

Comment:
Instead of the `CI` workflow calling `rockspec-names` action, I refactored it to take the relevant names as inputs. This is because actions can't call reusable workflows.